src/App.tsx

```diff
@@ -1,26 +1,27 @@
 import React from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useContext } from 'react';
 
 import Navbar from './components/Navbar/Navbar';
 import Home from './components/Home/Home';
 import Films from './components/Films/Films';
 import Login from './components/Login/Login';
-import Footer from './components/Footer/Footer';
+import AuthContext from './components/Store/auth-context';
 
 import 'antd/dist/antd.css';
 import './App.css';
 
 function App() {
+  const authCtx = useContext(AuthContext);
+  const isLoggedIn = authCtx.isLoggedIn;
   return (
     <Router>
-      <Navbar /> <br />
+      <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
-        <Route path="/films" element={<Films />} />
-        <Route path="/login" element={<Login />} />
-      </Routes>{' '}
-      <br /> <br />
-      <Footer />
+        {isLoggedIn && <Route path="/films" element={<Films />} />}
```

Comment:
Not a good option to check every route this way, please create a PrivateRoute component where you can check if the user is logged in or not. Please use custom Hook useAuth where all these checks of logged in user can happen